### ISSUE TITLE
Fix mapping of Mic to player

### ISSUE THREAD

**Issue by Michri** [Vocaluxe/Vocaluxe#572]
It will be really helpful, having the possibility of assigning a dedicated microphone to each user, so it will never change - useful especially, if amount of mics = amount of players. Then people do not need to run around and find the correct mic.
This will be really useful - independent to Corona.

**Comment by daggeg** [comment]
I suppose you for the challenge mode? If so I agree.

**Comment by Michri** [comment]
It might make sense maybe everywhere. When defining players, mic could be mapped to a player as well.

**Comment by daggeg** [comment]
I’m not sure what you mean, do you mean everywhere as in all party modes? When you just choose a song to sing this is not a problem, player one uses the mic dedicated for player one and so forth.

**Comment by Michri** [comment]
For a single song it might be ok, to define it in the sing-process. Then users can decide Flor per see, when assigning the players.
But even here it isn’t necessary, if the Color’s have already assigned before. 
so in simple sing-mode you assign the players only.

**Comment by daggeg** [comment]
I’m not sure I follow, in simple sing-mode you only assign the players once? Unless you are more players than you’ve got microphones.

**Comment by Michri** [comment]
Yes, right, but when being in one of the party modes, mics are changing and you need to hand over the mice potentially in each round.
So if you have same amount of mics and players, you can map mic to player 1:1 and keep this connection for all modes (simple sing mode or party mode). If you have more players than mics, you can still define a constant mapping, but of course some players might need to hand over the mics, but it will be always the same people sharing one mic color. 
To not break the consistancy, if might make sense to consider thre mic-player-mapping in the simple singing mode as well.

As I see:
1. define mic settings (each mic should be labeled with mic... instead of player, as the player is not always the same for this mic-color)
![1  define Mic settings](https://user-images.githubusercontent.com/20970069/149672719-4d447c79-c33d-42c6-8fc6-64bcb6b22c66.png)

2. configuring profile per player incl. mic
![2  define player profiles](https://user-images.githubusercontent.com/20970069/149672733-87a01320-904f-49e5-b33b-d8652852602e.png)

3. start playing 
    like simple singing (main menu "sing" > select a song and start singing)
    or select one of the pary-modes and start playing
    but keep always the same mic for a user
    ![Play-modes](https://user-images.githubusercontent.com/20970069/149672835-3e4e34ea-c173-4c88-9e50-d8e3735190e4.png)
